client/src/pages/dashboard.tsx

```diff
@@ -0,0 +1,80 @@
+import { format, fromUnixTime, getUnixTime } from 'date-fns';
+import { DashboardLayout } from '../components/DashboardLayout';
+import { Overview } from '../components/Overview';
+import { DashboardData, PriceResult } from '../types/FileData';
+import { Container } from '../ui/Container';
+
+interface DashboardProps {
+  statsData: DashboardData;
+}
+
+const Dashboard = ({ statsData }: DashboardProps) => {
+  return (
+    <Container
+      css={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '$5',
+        py: '$5',
+
+        '@lg': {
+          gap: '$20',
+          flexDirection: 'row',
+          py: '$10',
+          height: '100vh',
+        },
+
+        '@xl': {
+          gap: '128px',
+        },
+      }}
+    >
+      <DashboardLayout />
+      <Overview statsData={statsData} />
+    </Container>
+  );
+};
+
+export async function getStaticProps() {
+  const res = await fetch(process.env.DASHBOARD_API_URL!);
+  const stxStats = await res.json();
+
+  const priceRes = await fetch(process.env.STX_PRICE_API_URL!);
```

Comment:
As the coingecko API does not have a staging URL might be better to move it to the config file instead of the .env file, the value is never changing so will be easier for new devs to setup the client if this env variable is not needed :)

Reply:
Sure, that makes sense :) Would it be best to just extend the endpoint in the code like dashboard example above? Or just store the entire urls separately?

Reply:
Extending will work fine here :)